resources/fields/group_62af799fa840b.json

```diff
@@ -83,6 +99,41 @@
                     "prepend": "",
                     "append": "",
                     "maxlength": ""
+                },
+                {
+                    "key": "field_6347e6756cb13",
+                    "label": "Theme: Media Settings",
+                    "name": "",
+                    "type": "tab",
+                    "instructions": "",
+                    "required": 0,
+                    "conditional_logic": 0,
+                    "wrapper": {
+                        "width": "",
+                        "class": "",
+                        "id": ""
+                    },
+                    "placement": "left",
+                    "endpoint": 0
+                },
+                {
+                    "key": "field_6347e6d46cb14",
+                    "label": "Generate WEBP",
+                    "name": "webp_images",
+                    "type": "true_false",
+                    "instructions": "The module generates the webp version of the uploaded images. Please remember that it is not fully supported in modern browsers. It also requires special entries added in .htaccess.",
+                    "required": 0,
+                    "conditional_logic": 0,
+                    "wrapper": {
+                        "width": "",
+                        "class": "",
+                        "id": ""
+                    },
+                    "message": "",
+                    "default_value": 1,
```

Comment:
Shouldn't we default this to `false` (0)?
The `WEPB->isActive()` defaults to `false` as well as [getting the option during initialization in line 20](https://github.com/coditive-com/firestarter-theme/pull/40/files#diff-6b62743c4f27bf0139a9184302645266d00f8cff5a29f199730a3a20610eb4c0R20)

Reply:
Imho we should - just tested on the new project and while going to options this seemed to be "active" the WEBP wasn't being generated because the option was _probably_ not set yet.